app/assets/javascripts/discourse/app/services/composer.js

```diff
@@ -567,33 +572,6 @@ export default class ComposerService extends Service {
     this.close();
   }
 
-  @action
-  async openComposer(options, post, topic) {
```

Comment:
This function is very strangely named, especially given that we also have an `open` function. The only difference is that this one will prepend "Continued discussion from ..." to the composer.

To reduce confusion, I've added a 'prependText' option to the standard `open` function, and moved the "Continued discussion from..." text generation to the `<ComposerActions` component (the only place that was actually calling this function)